shared/src/commonTest/kotlin/com/tajemniktv/tajsos/ui/MainViewModelTest.kt

```
@@ -94,8 +96,10 @@ class MainViewModelTest {
         override suspend fun updateNode(node: NodeEntity) {
         }
 
+
         override suspend fun deleteNode(node: NodeEntity) {
         }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)